### ISSUE TITLE
Include @babel/plugin-proposal-class-properties  in @babel/preset-typescript

### ISSUE THREAD

**Issue by dilyanpalauzov** [babel/babel#12030]
I want to use Babel 7.11.5 only to strip TypeScript declarations, as fast alternative to running `tsc`.  With X.ts:
```typescript
class X {
    #y(n: number) {
      console.log(n + 2);
    }
}
```
This fails:
```
babel --presets @babel/preset-typescript --extensions '.ts' X.ts
SyntaxError: X.ts: Support for the experimental syntax 'classPrivateProperties' isn't currently enabled (2:5):

  1 | class X {
> 2 |     #y(n: number) {
    |     ^
  3 |       console.log( n + 2);
  4 |     }
  5 | }

Add @babel/plugin-proposal-class-properties (https://git.io/vb4SL) to the 'plugins' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-class-properties (https://git.io/vb4yQ) to the 'plugins' section to enable parsing.
  at Object._raise ...
  loc: Position { line: 2, column: 4 },
  pos: 14,
  missingPlugin: [ 'classPrivateProperties', 'classPrivateMethods' ],
  code: 'BABEL_PARSE_ERROR'
}
```
This works:
```bash
babel --plugins @babel/plugin-syntax-class-properties --presets @babel/preset-typescript --extensions '.ts' X.ts
```

• Extend `babel/cli --presets @babel/preset-typescript` or @babel/plugin-transform-typescript to only strip the TypeScript declarations and leave the rest as it is, so that adding `@babel/plugin-syntax-class-properties` is not necessary for the stripping.

**Comment by dilyanpalauzov** [comment]
Do I see correctly, that https://github.com/babel/babel/pull/13232 enables  @babel/plugin-syntax-class-properties by default (no-op), so that the typescript-preset can now be used just to strip the type annotations, and it understands this convoluted syntax?

**Comment by dilyanpalauzov** [comment]
This is implemented in 7.14.0.
